### PR TITLE
todo-app-list

### PR DESCRIPTION
**Pull request by letarulinda**
In this project, I implemented the following .
- Set up a new project with webpack that is based on the webpack.
- Create an index.html file .
- Create an index.js file and set an array of some simple to do tasks (array of objects). Each task object should contain three keys:
description [string].
completed [bool].
index: [number].
- Write a function to iterate over the tasks array and populate an HTML list item element for each task.
- Create a style.css and set rules for the To Do List.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+
+
+const formContainer = document.getElementById('form')
+const inputElement = document.getElementById('input-element');
+const myContainer = document.getElementById('items-container');
+const clearSelectedList = document.getElementById('footer-a')
+
+
+
```
Comment (KDlamini) [inline src/index.js:80]: The `index.js` file should have an array of some simple *to do tasks* (array of objects) with each object containing three keys - description \[`string`\], completed \[`bool`\], index: \[`number`\]. 

For example:
```
const tasks = [
  {
    description: 'An example description in string',
    completed: false,
    index: 10,
  },
];
```

- Kindly make sure that you have a few sample tasks in the array (at least 3 preferably) and they must be displayed in ascending order of the index. Currently, there are no sample tasks on the page.

image

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @letarulinda 🙋‍♀️,  

Great job so far! 💯 
There are some issues that you still need to work on to improve your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 

✔️  Webpack is configured correctly
✔️  There's a function for rendering data dynamically
✔️  UI follows design specifications
✔️  No linter errors

## Required Changes ♻️

**Kindly look through the comments below 👇 for required changes**

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (Abenezer-Tilahun) [reply src/index.js:80]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @letarulinda   ,
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Professional  README
✔️ Good Commit Messages


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**